exetera/core/field_importers.py

```diff
@@ -114,6 +114,16 @@ def __init__(self, create_day_field=False, create_flag_field=False):
 #============= Field Importers ============
 
 class CategoricalImporter:
+    """
+    Importer for Categorical field.
```

Comment:
Please say something about how the importers are used and whether end users are expected to use them directly for each importer